src/_includes/card.html

```diff
@@ -0,0 +1,13 @@
+{% assign gallery = collections.gallery | sortByPageOrder%}
+{% for item in gallery %}
+<li class="card">
+    <a href="{{ item.url }}">
+        <img src="{{ item.data.image }}" alt="">
```

Comment:
Note that you'd want to include `alt` text in your front matter for the files in the `/gallery/` so that this isn't left blank.